### ISSUE TITLE
Server stops responding to requests when Sass files are modified

### ISSUE THREAD

**Issue by vwong** [DylanPiercey/spawn-server-webpack-plugin#36]
I've been using this plugin in a project based on https://github.com/marko-js/examples/tree/master/examples/webpack-express

Recently, I upgraded from 6.2.0 to 6.2.1 and noticed that the server does not load correctly under some circumstances, but is ok in others.

When I first start the process with `webpack serve`, everything loads fine. However, if I make changes to Sass files, the browser is signalled to refresh the page, but the server thereafter never responds anymore, even with a different browser instance (to rule out a problem with the browser) - indicating that the server is not listening correctly anymore despite having the appropriate port open, or something along those lines.

Oddly, changes to `*.marko` and `*.js` files are reloaded fine.

The file/folder naming convention I'm using is
```
component.js
index.marko
style.scss
components/
  child/
    component.js
    index.marko
    style.scss
```

I'm using on Node 16.15.0 (Mac OS 12.3.1): 
```
    "sass": "1.52.1",
    "sass-loader": "^13.0.0",
    "webpack": "^5.72.1",
    "webpack-bundle-analyzer": "4.5.0",
    "webpack-cli": "^4.9.2",
    "webpack-dev-server": "^4.9.0",
    "webpack-node-externals": "^3.0.0"
```

(that's basically all the latest packages as of today 2022-05-29)

**Comment by vwong** [comment]
@DylanPiercey Any idea what the problem might be? I had a look at the diff between 6.2.0 and 6.2.1, but can't work out what caused the regression.

**Comment by DylanPiercey** [comment]
@vwong could you setup a reproduction repo? I just tried setting up the `webpack-express` example and adding a top level `.css` file and it doesn't seem to have this behavior. Wondering if there's something specific to `scss` maybe?

**Comment by vwong** [comment]
Ok, I'll try to set one up soonest possible

**Comment by vwong** [comment]
@DylanPiercey https://github.com/vwong/spawn-server-webpack-plugin-error reproduces the error. 
* Run `npm run dev` to start, open http://localhost:15015
* Notice the background color
* Update `src/pages/home/components/hero/style.scss` to change the background colour
* See that the browser is trying to reload but gets stuck
* Try opening using a different browser, and notice that gets stuck too
* Try changing a `*.ts`, or `*.marko` file, the server + browser reloads fine.
* Downgrade to `spawn-server-webpack-plugin@6.2.0`, everything works fine, including changing `*.scss` files.

Thanks for looking into this!

**Comment by DylanPiercey** [comment]
@vwong thanks for the reproduction. This should be fixed in `6.2.2`!

**Comment by vwong** [comment]
You. Are. Amazing!
